leakcanary-android-core/src/main/res/values/leak_canary_strings.xml

```diff
@@ -18,12 +18,12 @@
   <string name="leak_canary_about_title">About LeakCanary</string>
   <string name="leak_canary_about_message"><![CDATA[<a href="https://github.com/square/leakcanary">LeakCanary</a>
   is a memory leak detection library for Android, created by
-  <a href="https://twitter.com/Piwai">Pierre-Yves Ricau</a> and open sourced by Square.<br><br>
+  <a href="https://twitter.com/Piwai">Pierre-Yves Ricau</a> and open sourced by <a href="https://square.github.io">Square</a>.<br><br>
   We welcome contributions from the community, please do not hesitate to
   <a href="https://github.com/square/leakcanary/issues">report an issue</a> or open a pull request!<br>]]></string>
-  <string name="leak_canary_class_has_leaked">%1$s leaked</string>
+  <string name="leak_canary_class_has_leaked">%1$s Leaked</string>
```

Comment:
Why is this changing?

Reply:
Just Title Cased since it looked liked a title in the view; can revert if you want.